tests/lora/test_lora_layers_peft.py

```diff
@@ -817,6 +818,226 @@ def test_simple_inference_with_text_unet_multi_adapter(self):
                 "output with no lora and output with lora disabled should give same results",
             )
 
+    def test_simple_inference_with_text_unet_block_scale(self):
```

Comment:
Since the test has `text` inside of it, let's also target some text encoder layers in the `scales`.